src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,58 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+        Background:
+            Given a school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels       | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | [ "Tenth grade" ] | [ "School" ]                    |
+              And there is a valid AcademicWeek
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week One       | 2024-05-15 | 2024-05-15 | 2                      |
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by valid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate={beginDate}"
+             Then it should respond with 200
+              And response contains "weekIdentifier: "string" "
+
+        @ignore
+        Scenario: Ensure clients can't GET information when querying by invalid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=024-04-09"
+             Then it should respond with 400
+              And response body
+                  """
+                  {"detail":"Data validation failed. See 'validationErrors' for details.","type":"urn:ed-fi:api:bad-request:data","title":"Data Validation Failed","status":400,"correlationId":null,"validationErrors":{"$.beginDate":["The value '<beginDate>' is not valid for BeginDate."]}}
+                  """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by date out of specified dates
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=1970-04-09"
+             Then it should respond with 200
+              And response body is an empty array
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by begin date and end date
+            Given there is an invalid Begin date
```

Comment:
I think the Given can be removed here; and what exactly do you mean by invalid date? I'm thinking invalid as non-valid date format.

Reply:
The correct way to execute this scenario is to use dates that are not part of the data

Reply:
If that's the case I think we can add the values in the url, and you can add a comment mentioning the expected dates